<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSpectrogramVizualizer</class>
 <widget class="QWidget" name="QSpectrogramVizualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSpectrogramViewer" name="spectrogram_viewer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSoundPlayer" name="sound_player" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Spectrogram</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpectrogramOptions" name="spectrogram_options" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Image</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QImageOptions" name="image_options" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSoundPlayer</class>
   <extends>QWidget</extends>
   <header>pysoundplayer.gui.QSoundPlayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QImageOptions</class>
   <extends>QWidget</extends>
   <header>pysoundplayer.gui.QImageOptions.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QSpectrogramViewer</class>
   <extends>QWidget</extends>
   <header>pysoundplayer.gui.QSpectrogramViewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QSpectrogramOptions</class>
   <extends>QWidget</extends>
   <header>pysoundplayer.gui.QSpectrogramOptions.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
